src/wifi/model/periodic-dmg-wifi-scheduler.cc

```diff
@@ -300,31 +292,29 @@ PeriodicDmgWifiScheduler::UpdateAvailableSlots (uint32_t startAlloc, uint32_t en
   uint32_t startSlot;
   uint32_t endSlot;
 
-  for (auto it = m_availableSlots.begin(); it != m_availableSlots.end(); ++it)
+  for (auto it = m_availableSlots.begin (); it != m_availableSlots.end (); ++it)
```

Comment:
does this code work even without passing the iterator by ref?

Reply:
it didn't even compile with `auto&` as you suggested. This version compiles and works (usual test, checking the allocations)